app/src/main/java/com/onboarding/parkingsystemjava/mvp/presenter/SetParkingPresenter.java

```diff
@@ -0,0 +1,35 @@
+package com.onboarding.parkingsystemjava.mvp.presenter;
+
+import android.util.Log;
+
+import com.onboarding.parkingsystemjava.fragment.SetParkingLotsDialog;
+import com.onboarding.parkingsystemjava.mvp.contract.SetParkingContract;
+
+public class SetParkingPresenter implements SetParkingContract.SetParkingPresenter {
+    private SetParkingContract.SetParkingView view;
+    private SetParkingLotsDialog.OnInputListener inputListener;
+    private static final String TAG = "SetParkingPresenter";
+
+    public SetParkingPresenter(SetParkingLotsDialog.OnInputListener inputListener, SetParkingContract.SetParkingView v) {
+        this.view = v;
```

Comment:
do not use v as name, put a more representative name pls

Reply:
done